integration-cli/docker_api_containers_test.go

```diff
@@ -432,7 +432,7 @@ func (s *DockerSuite) TestGetStoppedContainerStats(c *check.C) {
 	}()
 
 	// allow some time to send request and let daemon deal with it
-	time.Sleep(1 * time.Second)
+	time.Sleep(time.Second)
```

Comment:
But yeah ``1 * time.Second`` makes it easier to change after so.. I'll make the inverted change :wink:.